app/src/main/java/tech/relaycorp/letro/account/model/Account.kt

```diff
@@ -27,6 +27,7 @@ data class Account(
     val veraidPrivateKey: ByteArray,
     val veraidMemberBundle: ByteArray? = null,
     val awalaEndpoint: String? = null,
```

Comment:
For consistency with the field below, I'd suggest adding the `Id` suffix:

```suggestion
    val awalaEndpointId: String? = null,
```

Reply:
Done